README.md

```diff
@@ -1,8 +1,68 @@
 
+### 🚨 Security review status: 
 
-#### 🚨 Security Review Status: Contracts frozen for audit as of commit [003e66d29ed40cceb2632655202f17ea1c0a2bb6](https://github.com/1Hive/dandelion-org/tree/003e66d29ed40cceb2632655202f17ea1c0a2bb6)
+Contracts frozen for audit as of commit [003e66d29ed40cceb2632655202f17ea1c0a2bb6](https://github.com/1Hive/dandelion-org/tree/003e66d29ed40cceb2632655202f17ea1c0a2bb6)
 
 The code in this repo has not been audited.
 
+## Overview
+
+1Hive's Dandelion Organization template is intended to help facilitate collaboration by providing an organization structure that makes it easy for contributors easily part ways when disagreements occur. By guaranteeing that participants can exit if they disagree with the decisions other members are making, dandelion organizations are more easily able to attract capital investment.
+
+The dandelion organization template consists of the Agent (optional), Finance, and Token Manager apps maintained by Aragon One. As well as the following applications developed and maintained by 1Hive:
+
+- [Redemptions](https://github.com/1Hive/redemptions-app): Allows users to manage a list of eligible assets held within an organizations Vault and allow members of the organization to redeem (burn) organization token in exchange for a proportional amount of the eligible assets.
+- [Token Request](https://github.com/1Hive/token-request-app): Allows users to propose minting tokens in exchange for a payment to the organization, subject to the approval of existing members.
+- [Time Lock](https://github.com/1Hive/time-lock-app): Allows an organization to require users to lock a configure amount of tokens for a configurable amount of time in order to forward an intent.
+- [Dandelion Voting](https://github.com/1Hive/dandelion-voting-app) An enhanced version of Aragon One's voting app which implements an ACL Oracle which allows an organization to configure permissions that restrict actions based on whether an address has recently voted Yes.
+
+## Permissions
+
+| App               | Permission             | Grantee                 | Manager          | ACL Oracle       |
+| ----------------- | ---------------------- | ----------------------- | ---------------- | ---------------- |
+| Kernel            | APP_MANAGER            | Dandelion Voting        | Dandelion Voting | None             |
+| ACL               | CREATE_PERMISSIONS     | Dandelion Voting        | Dandelion Voting | None             |
+| EVMScriptRegistry | REGISTRY_MANAGER       | Dandelion Voting        | Dandelion Voting | None             |
+| EVMScriptRegistry | REGISTRY_ADD_EXECUTOR  | Dandelion Voting        | Dandelion Voting | None             |
+| Dandelion Voting  | CREATE_VOTES           | Time Lock               | Dandelion Voting | None             |
+| Dandelion Voting  | MODIFY_QUORUM          | Voting                  | Dandelion Voting | None             |
```

Comment:
The use of "Voting" rather than "Dandelion Voting" seems inconsistent here.

Reply:
Good catch